cpp/basix/finite-element.cpp

```diff
@@ -216,6 +219,9 @@ basix::FiniteElement basix::create_element(element::family family,
                                            element::dpc_variant dvariant,
                                            bool discontinuous)
 {
+  if (family == element::family::custom)
+    throw std::runtime_error("Cannot create a custom element directly. Try "
```

Comment:
Curly braces around the multi-line statement would help readability here. (And below)